src/repr/src/adt/interval.rs

```diff
@@ -287,7 +287,8 @@ impl Interval {
             | DateTimeField::Decade
             | DateTimeField::Milliseconds
             | DateTimeField::Microseconds => {
-                unreachable!(format!("Cannot truncate interval by {}", f))
```

Comment:
Shouldn't this become `unreachable!("Cannot truncate interval by {}", f)`?

Reply:
Yes! Thanks :)